eraser-map.yaml

```diff
@@ -902,6 +902,13 @@ layers:
                 priority: 1
                 sprite: capital-xl
                 size: 18px
+    route_icon:
+        data: { source: route_icon }
+        draw:
+            icons:
+                priority: 1
+                sprite: airport
+                size: 18px
```

Comment:
Yup will do... sorry I accidentally merged this PR, will create another PR with `collide: false` and `transition` fixes.